02-oop-inheritance/js/script.js

```diff
@@ -0,0 +1,58 @@
+var Movie = function (title,year){
+    this.title = title;
+    this.year = year;
+    this.observerList = [];
```

Comment:
This should be defined here? What if I had multiple classes that should be observed?